lib/ui/pages/summary/summary_page.dart

```diff
@@ -8,10 +9,23 @@ class SummaryPage extends StatelessWidget {
 
   @override
   Widget build(BuildContext context) {
-    return BlocBuilder<SummaryBloc, SummaryState>(
+    final summaryBloc = context.read<SummaryBloc>();
+    final navigationBloc = context.read<NavigationBloc>();
+    return BlocConsumer<SummaryBloc, SummaryState>(
+      listener: (context, state) async {
+        if (state is SummaryNavigatePop) {
+          navigationBloc.add(NavigationPopEvent());
```

Comment:
BTW it's a bad pattern ))) When you open you big project in a month foe ex. it's hard to read: 
bloc.add(NewEvent(+))
bloc.add(NewEvent(-))

much better:
bloc.increment(NewEvent(+))
bloc.decrement(NewEvent(-))
It's just a dummy example )

Reply:
if i understood you correctly, your idea is to add public methods in navigation bloc like next example? 
```
class NavigationBloc extends Bloc<NavigationEvent, NavigationState> {
  NavigationBloc() : super(const NavigationInitial(routeStack: [])) {
    on<NavigationOpenDateOfBirthPage>(_onNavigationOpenDateOfBirthPage);
    on<NavigationOpenSummaryPage>(_onNavigationOpenSummaryPage);
    on<NavigationPopEvent>(_onNavigationPopEvent);
    on<NavigationOpenChoicePage>(_onNavigationOpenChoicePage);
  }

  void openDateOfBirthPage() {
    add(NavigationOpenDateOfBirthPage());
  }

  void openSummaryPage() {
    add(NavigationOpenSummaryPage());
  }

  void openChoicePage() {
    add(NavigationOpenChoicePage());
  }

  void popNavigation() {
    add(NavigationPopEvent());
  }
 ... rest of the original code
 }
 ```